frontends/web/public/btcdirect/fiat-to-coin.html

```diff
@@ -10,104 +10,123 @@
-    (function (btc, d, i, r, e, c, t) {
-      btc[r] = btc[r] || function () {
-        (btc[r].q = btc[r].q || []).push(arguments)
-      };
-      c = d.createElement(i);
-      c.id = r; c.src = e; c.async = true;
-      c.type = 'module'; c.dataset.btcdirect = '';
-      t = d.getElementsByTagName(i)[0];
-      t.parentNode.insertBefore(c, t);
-    })(window, document, 'script', 'btcdirect',
-      mode === 'production'
-      ? 'https://cdn.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
-      : 'https://cdn-sandbox.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
-    );
-
-    btcdirect('init', {
-      token: apiKey,
-      debug: mode === 'debug',
-      locale: window.frameElement?.dataset.locale || 'en-US',
-      theme: window.frameElement?.dataset.theme || 'light',
-    });
-
-    // fiat to coin order
-    btcdirect('wallet-addresses', {
-      addresses: {
-        address,
-        currency,
-        id: 'BitBox',
-        walletName: 'BitBox'
-      }
-    });
-
-    btcdirect('set-parameters',
-      mode === 'production' ? {
-        baseCurrency: currency,
-        fixedCurrency: true,
-        quoteCurrency,
-        // paymentMethod: any of 'bancontact', 'bankTransfer', 'creditCard', 'giropay', 'iDeal', 'sofort', 'applepay'
-        showWalletAddress: false,
-      } : {
-        baseCurrency: currency,
-        fixedCurrency: true,
-        paymentMethod: 'sofort', // sandbox currently only supports sofort payment method
-        quoteCurrency,
-        showWalletAddress: false,
+          ? 'https://cdn.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
+          : 'https://cdn-sandbox.btcdirect.eu/fiat-to-coin/fiat-to-coin.js'
+        );
+
+        btcdirect('init', {
+          token: apiKey,
+          debug: mode === 'debug',
+          locale: window.frameElement?.dataset.locale || 'en-GB',
+          theme: window.frameElement?.dataset.theme || 'light',
+        });
+
+        // fiat to coin order
+        btcdirect('wallet-addresses', {
+          addresses: {
+            address,
+            currency,
+            id: 'BitBox',
+            walletName: 'BitBox'
+          }
+        });
+
+        btcdirect('set-parameters',
+          mode === 'production' ? {
+            baseCurrency: currency,
+            fixedCurrency: true,
+            quoteCurrency,
+            // paymentMethod: any of 'bancontact', 'bankTransfer', 'creditCard', 'giropay', 'iDeal', 'sofort', 'applepay'
+            showWalletAddress: false,
+          } : {
+            baseCurrency: currency,
+            fixedCurrency: true,
+            paymentMethod: 'sofort', // sandbox currently only supports sofort payment method
+            quoteCurrency,
+            showWalletAddress: false,
+          }
+        );
+
+        window.addEventListener('btcdirect-embeddable-fiat-to-coin-order-confirmed', (event) => {
+          console.log('btcdirect-embeddable-fiat-to-coin-order-confirmed', event);
+          // Handle the event
```

Comment:
Here you could directly post a message to pass the order confirmation, maybe? And leave the TODO in the handler inside btcdirect.tsx. No strong opinion, tho

Reply:
I only tested once, this event is fired after the user clicked proceed to payment and not really useful this way, I'll just remove.

Reply:
removed in https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3152